static/editor.js

```diff
@@ -107,6 +108,13 @@ define(function (require) {
                 this.eventHub.emit('modifySettings', {
                     compileOnChange: !this.settings.compileOnChange
                 });
+                new Alert().notify('Compile on code change has been toggled ' + (this.settings.compileOnChange ? 'ON' : 'OFF'), {
+                    "group": "togglecompile",
+                    "noCollapse": false,
+                    "alertClass": this.settings.compileOnChange ? "notification-on" : "notification-off",
+                    "noAutoDissmis": false,
```

Comment:
Typo/spelling mistake here and below: "Dismiss" one 's' at the start and two at the end. English spellings are dumb :)